package.json

```diff
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/electrode-io/ern-navigation#readme",
   "dependencies": {
-    "ernnavigation-api": "^1.2.5",
+    "ernnavigation-api": "^1.4.0",
```

Comment:
Actually, `^1.2.5` _also_ matches 1.4.0 (if that is the intended update) - So to change to that version, only `yarn.lock` would need to be updated (which will also resolve the current error on CI).